javatests/arcs/core/entity/HandleManagerTestBase.kt

```diff
@@ -436,9 +438,9 @@ open class HandleManagerTestBase {
         // Remove the entity from the collection.
         writeEntityHandle.remove(entity1)
 
-        // Reference should be dead.
-        // TODO(b/152436411): The reference should dereference to null, but in fact dereferences to
-        // a RawEntity full of nulls.
+        // Reference should be dead. (Removed entities currently aren't actually deleted, but
+        // instead are "nulled out".
+        assertThat(storageReference.dereference()).isEqualTo(createNulledOutPerson("entity1"))
```

Comment:
yes this is a bit strange. would you prefer having "null" instead? maybe we could add support for that to refmodestore